apps/website/vitest.config.ts

```diff
@@ -1,11 +1,14 @@
 import { fileURLToPath } from "node:url";
 import { defineConfig } from "vitest/config";
-import "./build-scripts/env.mjs";
 
-export default defineConfig({
+import "./build-scripts/env";
+
+const config = defineConfig({
   resolve: {
     alias: {
       "@": fileURLToPath(new URL("./src", import.meta.url)),
     },
   },
 });
+
+export default config;
```

Comment:
Out of curiosity, why declare a const and then export it, rather than export default like we do in a few other places?

Reply:
So the eslint rule [import/no-anonymous-default-export](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-anonymous-default-export.md) enabled through the next eslint config does not allow exporting an object literal without a name as default. So for config files where there is not a `defineConfig()` call, we have to assign it to a var first. I think this also works better for declaring the type with jsdocs. I felt like its cleaner if we do the same for the configs where the config goes through `defineConfig()` too, but i dont have strong feelings.

For the anonymous exports I kinda agree and understand the rule, cause it makes it easier to (visually) grep import and export. I do like named exports for that reason too. But again no strong feelings.

Reply:
Could we update the tailwind + lint-staged configs to follow the same pattern for consistency?

Reply:
Interesting that the rule does not complain about those. Yeah we can do that for sure